docs/src/managing-packages.md

```diff
@@ -426,3 +426,13 @@ To run a typical garbage collection with default arguments, simply use the `gc`
 ```
 
 Note that only packages in `~/.julia/packages` are deleted.
+
+## Offline Mode
+
+
+In offline mode Pkg tries to do as much as possible without connecting
+to internet. For example, when adding a package Pkg only considers
+versions that are already downloaded in version resolution.
+
+To work in offline mode across Julia sessions you can
```

Comment:
Maybe mention `Pkg.offline` instead?  That's easier to use than setting an environment variable.